### PR TITLE
Added Instagram Logo

### PR DESCRIPTION
**Pull request by JoshBowdenConcepts**
## Summary

Added Instagram Logo to MinimalFooter Social Links

## List of notable changes:

- **added** Instagram logo to MinimalFooter social links

## What should reviewers focus on?

- Making sure the logo is the right size and color.

## Steps to test:

1. Open the MinimalFooter component in CI-deployed preview environment
2. Go to MinimalFooter story in Storybook
3. Verify that Instagram social link behaves as described in the following issue.

## Supporting resources (related issues, external links, etc):

- closes https://github.com/primer/brand/issues/266

## Contributor checklist:

- [x] All new and existing CI checks pass
- [x] Tests prove that the feature works and covers both happy and unhappy paths
- [x] Any drop in coverage, breaking changes or regressions have been documented above
- [x] New visual snapshots have been generated / updated for any UI changes
- [x] All developer debugging and non-functional logging has been removed
- [x] Related issues have been referenced in the PR description

## Reviewer checklist:

- [ ] Check that pull request and proposed changes adhere to our [contribution guidelines](../../CONTRIBUTING.md) and [code of conduct](../../CODE_OF_CONDUCT.md)
- [ ] Check that tests prove the feature works and covers both happy and unhappy paths
- [ ] Check that there aren't other open Pull Requests for the same update/change

## Screenshots:

> Please try to provide before and after screenshots or videos

<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">

 </td>
<td valign="top">

</td>
</tr>
</table>

### CODE REVIEW

**Review by rezrah — COMMENTED** [review]
FYI The visual snapshots will need to be updated. I've added the label for you just now.

File: packages/react/src/MinimalFooter/MinimalFooter.tsx [hunk 1]
```diff
@@ -212,6 +212,14 @@ function SocialLogomarks({socialLinks, logoHref}: SocialLogomarksProps) {
       icon: 'https://github.githubassets.com/images/modules/site/icons/footer/tiktok.svg',
       iconWidth: 18,
       iconHeight: 18
+    },
+    {
+      name: 'instagram',
```
Comment (rezrah) [inline packages/react/src/MinimalFooter/MinimalFooter.tsx:217]: Could you also update the doc prop table please to reference Instagram.

Screenshot 2023-05-30 at 09 30 56

Comment (JoshBowdenConcepts) [reply packages/react/src/MinimalFooter/MinimalFooter.tsx:217]: This has been resolved in the following [SHA](https://github.com/primer/brand/pull/282/commits/916b5dbe64cb99e76ae8d4527aaeaf09f6434f9b)

Comment (JoshBowdenConcepts) [reply packages/react/src/MinimalFooter/MinimalFooter.tsx:217]: This has been resolved in the following [SHA](https://github.com/primer/brand/pull/282/commits/3c23a310137a9d2b481c10fb5b2400eb4f23feeb)

**Review by rezrah — APPROVED** [review]

**Pull request merged**